Modules/_dbmmodule.c

```diff
@@ -396,8 +396,9 @@ dbm__enter__(PyObject *self, PyObject *args)
 static PyObject *
 dbm__exit__(PyObject *self, PyObject *args)
 {
-    _Py_IDENTIFIER(close);
-    return _PyObject_CallMethodIdNoArgs(self, &PyId_close);
+    _dbm_state *state = PyType_GetModuleState(Py_TYPE(self));
+    assert(state != NULL);
+    return PyObject_CallMethodNoArgs(self, state->str_close);
```

Comment:
I think you can just use `_dbm_dbm_close_impl` instead of going though the Call API; ditto for gdbm :)

Reply:
> I think you can just use _dbm_dbm_close_impl instead of going though the Call API; ditto for gdbm :)

Sounds great, I will update it